src/framework/theme/components/chat/chat.component.ts

```diff
@@ -286,9 +298,9 @@ export class NbChatComponent implements OnChanges, AfterContentInit, AfterViewIn
   @ViewChild('scrollable') scrollable: ElementRef;
   @ContentChildren(NbChatMessageComponent) messages: QueryList<NbChatMessageComponent>;
   @ContentChild(NbChatFormComponent) chatForm: NbChatFormComponent;
+  @ContentChild(NbChatTitleDirective) templateTitle: NbChatTitleDirective;
```

Comment:
```suggestion
  @ContentChild(NbChatTitleDirective) titleTemplate: NbChatTitleDirective;
```
To conform to similar window feature